cartridge-cli.lua

```diff
@@ -192,30 +192,25 @@ end
 
 -- * ------------------------------ Messages ------------------------------
 
-local function print_and_flush(msg)
-    print(msg)
-    io.flush()
-end
-
 local function die(fmt, ...)
     local msg = "ERROR: " .. string.format(fmt, ...)
-    print_and_flush(colored_msg(msg, ERROR_COLOR_CODE))
+    log.error(colored_msg(msg, ERROR_COLOR_CODE))
     os.exit(1)
 end
 
 local function warn(fmt, ...)
     local msg = "WARNING: " .. string.format(fmt, ...)
-    print_and_flush(colored_msg(msg, WARN_COLOR_CODE))
+    log.warn(colored_msg(msg, WARN_COLOR_CODE))
 end
 
 local function info(fmt, ...) -- luacheck: no unused
     local msg = string.format(fmt, ...)
-    print_and_flush(colored_msg(msg, INFO_COLOR_CODE))
+    log.info(colored_msg(msg, INFO_COLOR_CODE))
 end
 
 local function debug(fmt, ...) -- luacheck: no unused
     local msg = string.format(fmt, ...)
-    print_and_flush(colored_msg(msg, DEBUG_COLOR_CODE))
+    log.info(colored_msg(msg, DEBUG_COLOR_CODE))
```

Comment:
debug -> info? But it's ok because by default we can't log in debug mode without box.cfg